x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/constants.ts

```diff
@@ -14,3 +14,10 @@ export const META_ENGINES_TITLE = i18n.translate(
   'xpack.enterpriseSearch.appSearch.metaEngines.title',
   { defaultMessage: 'Meta Engines' }
 );
+
+export const CREATE_AN_ENGINE_BUTTON_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engines.createAnEngineButton.label',
+  {
+    defaultMessage: 'Create an Engine',
+  }
+);
```

Comment:
I strongly disagree, based on my experience on the i18n team at Meetup, with re-using keys in different contexts just because their current English translations match.  One is the title of the view and the other is a CTA for a form. If I had another view that was creating engines and we wanted to re-use this as a generic "createEngineCTA.labal", I'd be amenable, but I think titles/button labels specifically are different context and that context matters a lot with localization, especially when translaters are just working from a big ol excel doc.. Safer to seperate these, even if it does add more work for localizers.

Reply:
Ahh that makes a ton of sense. We haven't previously been super properly labeling message types (label, title, etc.) according to Kibana's docs (https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types), but definitely should going forward. CC @JasonStoltz for info

Reply:
Thanks for linking this!